openshift/configmap.yaml

```diff
@@ -74,4 +75,9 @@ parameters:
   name: KAFKA_TIMEOUT_SECONDS
   required: true
   value: "30"
+- description: Directory for Prometheus metrics.
+  displayName: Masu Prometheus metrics directory
+  name: PROMETHEUS_DIR
+  required: true
+  value: '/tmp'
```

Comment:
Just want to double check that we want these metrics going to `/tmp`

Reply:
So we just need an existing directory, we don't need to really retain this data as the Prometheus data scraper asks for data ever 20 seconds or so.

Just picked `/tmp`? You think some other location would be better?

Reply:
Nope `/tmp` sounds good for that purpose.